docs/architecture/high-level-components.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+This document describes communication between Management Plane, Application and Runtimes. 
+
+## Components
+Management Plane is a central place which stores configuration about Applications and Runtimes and is responsible for propagating that information to Applications and Runtimes. 
+Management Plane plays crucial role in establishing trusted connection between Applications and Runtimes. 
+![](./assets/components-high-level.svg)
```

Comment:
Hmm, I think this diagram is a bit too complicated on the left side: Application <-> Management Plane relation. I would simplify it in the following way:

- Remove webhook from this diagram, as IMHO it should be as low level of abstraction as possible
- Remove "add runtime, add application, configure" verbs from Admin and replace it with something like "manage applications and runtimes", or even simpler "manage"
- Does app really configure Management Plane? That's what I read it from the diagram. The arrow should be on the opposite side, right? Because Management Plane passes configuration to Application and Management Plane set-ups trust between App and Runtime. 
- Sync between Agent and Management Plane is two way, right? So the arrows should be on both sizes of the line
-